### PR TITLE
Update the templates so users know the response to expect

### PR DESCRIPTION
**Pull request by kienstra**
## Changes
* Update the templates so users know the response to expect

## Testing instructions
Not needed

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -4,13 +4,4 @@ about: Request an enhancement, not a bug fix
...
-<!-- In as much detail as possible -->
-
-## Image
-<!-- Even a simple sketch of the feature would help -->
-
-## Use case
-<!-- What's your use case for this feature? Even if we can't add it, we might be able to address your case. -->
-
-## Additional information
+Thanks, but we're not adding new features.
```
Comment (kienstra) [inline .github/ISSUE_TEMPLATE/feature_request.md:7]: This isn't a secret, from a simple look at the [Changelog](https://wordpress.org/plugins/genesis-custom-blocks/#developers).

This way, we won't disappoint users when they request a feature we don't add (or even respond to).

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -4,6 +4,8 @@ about: Report a problem
 
 ---
 
+<!-- We'll respond to big bugs, but probably not smaller ones. -->
```
Comment (kienstra) [inline .github/ISSUE_TEMPLATE/bug_report.md:7]: Inspired by ACF's great support topic: https://wordpress.org/support/topic/free-support-expectations-and-response-times-7/

**Review by mike-day — APPROVED** [review]
This is another good idea, @kienstra. It's only fair to be upfront with users about expectation.

**Comment by kienstra** [comment]
Thanks so much, @mike-day!

**Pull request merged**